src/Raven.Server/Documents/TcpHandlers/SubscriptionConnection.cs

```diff
@@ -260,6 +297,10 @@ public static void SendSubscriptionDocuments(ServerStore server, TcpConnectionOp
                         using (tcpConnectionDisposable)
                         using (connection)
                         {
+                            connection._lastConnectionStats = new SubscriptionConnectionStatsAggregator(_connectionStatsId, null);
```

Comment:
please compare to ETL - IIRC we pass Interlocked.Increment(ref _connectionStatsId) as argument - so first items get 1 not 0 as id.

Reply:
done